packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -112,6 +112,16 @@
   display: flex;
   width: 100%;
   box-sizing: border-box;
+  &:before {
```

Comment:
Hmm I can give it a try. My only concern in using @mixin after-outline is this is adding an after and I need a before. When I add an after the icons disappear and borders appear on all the sub element as well. maybe I can create a @mixin before-outline... will try

My concern is really about scope. Like if we take the time to create the mixin should we take the time to migrate the other :befor-borders in the project?